### ISSUE TITLE
missing profile for ORS requests

### ISSUE THREAD

**Issue by mbogner** [VROOM-Project/vroom-frontend#69]
When using ORS requests need to use profile "driving-car" instead of default "car". Otherwise this results in a 400 response with the message that profile "car" doesn't exist which is correct because ORS named it "driving-car".

This fixed the issue for me though it is specific for ORS and fixes all cars to the same profile (which not adding one does as well):

```js
var vehicles = JSON.parse(JSON.stringify(dataHandler.getVehicles()));
for (let i = 0; i < vehicles.length; i++) {
    vehicles[i].profile = 'driving-car';
}
var input = {
    jobs: JSON.parse(JSON.stringify(dataHandler.getJobs())),
    shipments: JSON.parse(JSON.stringify(dataHandler.getShipments())),
    vehicles: vehicles,
    "options": {
        "g": true
    }
};
```

**Comment by jcoupey** [comment]
That's correct, we don't have a notion of `profile` in this frontend, it only works out of the box with OSRM, relying on the fact that `car` is the default there.

Not sure whether we should do anything about it, except maybe make that default explicit and easy to adjust in a config file somewhere.

**Comment by mbogner** [comment]
Having it documented and configurable would be great.

**Comment by jcoupey** [comment]
Happy to review a PR introducing a configurable default profile!

**Comment by mbogner** [comment]
Let's see if I find time :)
